### ISSUE TITLE
Add mipsle64

### ISSUE THREAD

**Issue by J0WI** [alpinelinux/docker-alpine#76]
https://github.com/docker-library/official-images/issues/6709

**Comment by tianon** [comment]
See https://github.com/alpinelinux/docker-alpine/issues/30

**Comment by J0WI** [comment]
Not sure if this is still valid: https://pkgs.alpinelinux.org/packages?arch=mips64

**Comment by ncopa** [comment]
It is valid, but mips support is fresh and we dont yet have it included in any edge release:
http://dl-cdn.alpinelinux.org/alpine/edge/releases/

**Comment by tianon** [comment]
From what I can tell, Alpine's `mips64` is big-endian, right?
(going by https://github.com/alpinelinux/aports/commit/d6fb4cc81870d5b1ba25c4addd9b804aa0186380; specifically `CONFIG_CPU_BIG_ENDIAN`)

So this won't be `mips64le`, but rather `mips64` (going by `GOARCH` conventions used for OCI platform specifications; I'm not sure whether our official-images builder can support big-endian as-is, but I believe it can).

**Comment by kaniini** [comment]
Correct, Alpine mips64 is big endian.  We are considering a mips64le port too though.

**Comment by XiaodongLoong** [comment]
We have communicated with alpine community. And there is a plan to support mips64el architectures. (https://lists.alpinelinux.org/~alpine/devel?search=mips64el).
